api/http/server_test.go

```diff
@@ -945,7 +945,7 @@ func testBzzTar(encrypted bool, t *testing.T) {
 
 	// check that the tag was written correctly
 	tag := srv.Tags.All()[0]
-	chunktesting.CheckTag(t, tag, 4, 4, 0, 4)
+	chunktesting.CheckTag(t, tag, 3, 3, 0, 0, 0, 3)
```

Comment:
we should check why the modified use of context in http/server results in different number of tags. a manifest is missing maybe?